astropy/io/misc/tests/test_yaml.py

```diff
@@ -148,7 +148,7 @@ def compare_coord(c, cy):
     assert list(c.representation_component_names) == list(
         cy.representation_component_names
     )
-    for name in c.representation_component_names:
+    for _ in c.representation_component_names:
```

Comment:
There must be a mistake in the the test. The next line should use `name`.

Reply:
I think that should be `for attr in c.representation_component_names:`.